lib/cli/src/generate-annotations.ts

```diff
@@ -0,0 +1,69 @@
+import path from 'path';
+import dedent from 'ts-dedent';
+import { writeFile, existsSync } from 'fs-extra';
+import { getAllPresets } from '@storybook/core-common';
+import { logger } from '@storybook/node-logger';
+
+import { viewLayers } from './view-layers';
+
+export const getAnnotationsFileContent = (presets: string[], replaceEsmWithCjs?: boolean) => {
+  const presetNames: string[] = [];
+  const presetImports: string[] = [];
+
+  let previewFile;
+  const [firstImport] = presets.slice(-1);
+  if (firstImport.indexOf('preview') !== -1) {
```

Comment:
This check isn't going to be good if we are calling the annotation files `.../preview.js`

Reply:
Fixed!